app/tasks/github_receiver.py

```diff
@@ -173,10 +173,11 @@ def _handle_jira_issue(self, project_key, issue_autocard,
             elif action == 'closed':
                 self._delete_issue_jira_issue_objects()
             else:
-                print('Unsupported event action.')
+                print('Unsupported event action: {0}'.format(action))
 
         elif scope == "pull_request":
-            label_names += [label['name'] for label in self.payload['issue']['labels']]
+            if 'issue' in self.payload
```

Comment:
I think this is missing a colon at the end.

Reply:
Good catch, resolved 👍